### DISCUSSION TITLE
Going from Orientation to Deck Request

### DISCUSSION THREAD

**Discussion by MeVall42** [bytedeck/bytedeck#1582] [Q&A]
Hello!

I recently completed the Bytedeck Orientation and am waiting for approval. I also used the map to preview the "request your deck" form and requested a trial deck. It has been more than the stated 48 hours, so I'm wondering if I might have done something wrong. Is there a way to tell whether or not a submission has gone through or has been seen?

Thanks!
Mela Vallentgoed

**Comment by tylerecouture** [comment]
Thanks for connecting here. I will look into it first thing tomorrow morning!

**Reply by MeVall42** [reply]
Thank you! :)
